packages/now-cli/test/integration.js

```diff
@@ -770,30 +770,6 @@ test('remove the wildcard alias', async t => {
   t.true(stdout.startsWith(goal));
 });
 
-test('ensure type and instance count in list is right', async t => {
```

Comment:
Why was this test removed? It seems like you should modify it to correctly test the list output.

Reply:
@styfle Ahh, I thought we could removed it since we are long longer have those field.

Will add it back checking for username instead 🙌